kaldb/src/test/java/com/slack/kaldb/logstore/search/LogIndexSearcherImplTest.java

```diff
@@ -37,8 +41,10 @@ public static void beforeClass() throws Exception {
   private void loadTestData(Instant time) {
     strictLogStore.logStore.addMessage(
         makeMessageWithIndexAndTimestamp(1, "apple", TEST_INDEX_NAME, time));
-    strictLogStore.logStore.addMessage(
-        makeMessageWithIndexAndTimestamp(2, "baby", "new" + TEST_INDEX_NAME, time.plusSeconds(1)));
+
+    // todo - re-enable when multi-tenancy is supported
```

Comment:
Please add the issue number to this class also so we can refer back to it.

Reply:
Done.